ngraph/src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -55,9 +55,9 @@ namespace
     ngraph::NodeVector get_outputs(const std::shared_ptr<ngraph::Node>& node)
```

Comment:
@postrational Can ONNX importer return OutputVector and don't use `GetOutputElement`?